From 3a64779ec5a89d38d64e07bca2b11b19e1882d7a Mon Sep 17 00:00:00 2001
From: Charles Duffy <charles@dyfis.net>
Date: Thu, 24 Sep 2009 09:00:24 +0100
Subject: [PATCH] Fix unitialized variable in qemudDomainDetachHostPciDevice()

* src/qemu_driver.c: initialize detach var

(cherry-picked from commit 580ad29288751234bee47ac9f6c04dac1dc529ea)

Fedora-patch: libvirt-fix-device-detach-typo3.patch
---
 src/qemu_driver.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/qemu_driver.c b/src/qemu_driver.c
index 7c7b985..550a59c 100644
--- a/src/qemu_driver.c
+++ b/src/qemu_driver.c
@@ -6126,7 +6126,7 @@ static int qemudDomainDetachHostPciDevice(virConnectPtr conn,
                                           virDomainObjPtr vm,
                                           virDomainDeviceDefPtr dev)
 {
-    virDomainHostdevDefPtr detach;
+    virDomainHostdevDefPtr detach = NULL;
     char *cmd, *reply;
     int i, ret;
     pciDevice *pci;
-- 
1.6.2.5

